source/contracts/Time.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity 0.4.18;
+
+import 'ITime.sol';
```

Comment:
I'm not a huge fan of leaving this to live in the root contracts folder, any reason to not put them in `libraries`?

Reply:
Generally the libraries folder is for contracts that aren't actually uploaded. I wouldn't be opposed to a new folder. We could make a corw folder and put all the stragglers in there or a utils folder and just put these two there

Reply:
Hmm... good point, I think we actually have code in the contract deployer that ignores the libraries folder.  I would prefer a folder for this kind of thing, possibly move `Augur`, `Controller` and `Controlled` in there too?  Those three contracts definitely feel disconnected from each other... okay, maybe the root is the best place until we start seeing enough stuff pop up to appropriately categorize.